### PR TITLE
[TT-9615] Bug: nesting of autosuggest component on input element

### PR DESCRIPTION
**Pull request by chaddgrimm**
This fixes the issue shown on the clip below:

https://github.com/user-attachments/assets/b96aa1c3-5563-46d1-b58a-d414177d7dfd

Additional changes:
- Upgraded cypress version
- Moved to port 8080

### CODE REVIEW

File: Dockerfile
Comment (imndaiga) [inline Dockerfile:1]: nit: can use [wp-cli](https://make.wordpress.org/cli/handbook/guides/quick-start/#practical-examples) to do the wp setup instead

File: test/database/sql/wordpress-5.7.2.sql
Comment (imndaiga) [inline test/database/sql/wordpress-5.7.2.sql:1]: for clarity, why are we committing the DBs? Does this give us any benefit besides reproducible state across test runs?

**Review by imndaiga — APPROVED** [review]
LGTM - small nit re: test workflows

File: Dockerfile
Comment (chaddgrimm) [reply Dockerfile:1]: I'll park this for the next one, but this looks good.

File: test/database/sql/wordpress-5.7.2.sql
Comment (chaddgrimm) [reply test/database/sql/wordpress-5.7.2.sql:1]: In a sense yes, the script file from what I understand is just there to help setup the initial state of our DB, i.e: shop item etc.

**Pull request merged**